### PR TITLE
MEN-7442: Update integration code for v0.2-update-modules

### PR DESCRIPTION
**Pull request by lluiscampos**

### CODE REVIEW

File: west.yml [hunk 1]
```diff
@@ -13,7 +13,8 @@ manifest:
       import: true
     - name: mender-mcu
       remote: mender
-      revision: v0.1-minimal
+      # TODO: set to v0.2-...
+      revision: main
```
Comment (lluiscampos) [inline west.yml:17]: This TODO will be done after:
* https://github.com/mendersoftware/mender-mcu/pull/71
* https://github.com/mendersoftware/mender-mcu/pull/73

**Review by larsewi — APPROVED** [review]
🚀

File: src/main.c [hunk 2]
```diff
@@ -118,6 +99,11 @@ main(void) {
     assert(MENDER_OK == mender_client_init(&mender_client_config, &mender_client_callbacks));
     LOG_INF("Mender client initialized");
 
+#ifdef CONFIG_MENDER_ZEPHYR_IMAGE_UPDATE_MODULE
+    assert(MENDER_OK == mender_zephyr_image_register_update_module());
```
Comment (vpodzime) [inline src/main.c:103]: This will be a no-op in a non-debug build. Something to keep in mind. :wink:

Comment (lluiscampos) [reply src/main.c:103]: I don't think so - I've stumble upon these with release builds with Zephyr OS defaults. The main difference is that it just aborts, while on debug it does log the assertion, line number etc.

**Pull request merged**